src/mount/mastercomm.cc

```diff
@@ -95,6 +95,11 @@ static bool disconnect;
 static time_t lastwrite;
 static int sessionlost;
 
+#ifdef _WIN32
+uint8_t *session_flags_ = NULL;
+int *mounting_uid_ = NULL, *mounting_gid_ = NULL;
```

Comment:
Please use one declaration per line and nullptr.
Also consider renaming the variables to gradually match de defined code style: mountingUid and mountingGid.

Reply:
Will be updated.

Reply:
Will be updated.

Reply:
I see it was already done.